### DISCUSSION TITLE
Confusion over chars in sets

### DISCUSSION THREAD

**Discussion by iandoug** [koeberlin/Latin-Character-Sets#9] [Ideas]
Hi

The Libertinus guys suggested I add your sets to my comparison page
https://www.keyboard-design.com/glyph-coverage-comparison.html
(Dev version also includes MUFI list)

Having some troubles getting the list of chars in each set.

PHP is misbehaving with for example LatinS_unicodes.txt.
However, a safer choice, like LatinS_hex.txt, appears to be missing ASCII etc.

Is there a file with each set with Unicode points... something like the tables in the readme?

Think I will just extract from the readme, but a set like that, computer-friendly, would be useful.

Thanks, Ian

**Comment by koeberlin** [comment]
Hi Ian,
thank you for adding the sets to your comparison page!
You’re right, ASCII is missing from the LatinS_hex.txt file, and it shouldn’t be.
I just updated it.
Cheers,
Christoph

**Reply by iandoug** [reply]
Thanks, I have updated my page to include your sets.

Need to figure out how to set the MIME type on the .tsv version.
